main.go

```diff
@@ -75,6 +76,11 @@ func main() {
 		}
 	})
 
+	r.POST("/exec/config", func(c *gin.Context) {
+		logrus.Debugf("handling /exec/config")
```

Comment:
Seems it can be removed since requests are logged by GIN

![Screenshot_20200505_123528](https://user-images.githubusercontent.com/5887312/81052883-ebdfb280-8ecc-11ea-99e0-91f02c292c6e.png)